deployment/chart/templates/config.yaml

```diff
@@ -3,25 +3,17 @@ kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-config
 data:
-  RAILS_ENV: "{{ .Values.ticketsEnvironment }}"
-  SMTP_ADDRESS: "{{ .Values.ticketsSMTPHost }}"
-  SMTP_PORT: "{{ .Values.ticketsSMTPPort }}"
-  SMTP_DOMAIN: "{{ .Values.ticketsSMTPDomain }}"
-  TICKETS_HOST: "{{ .Values.ticketsHost }}"
-  ACTION_MAILER_DEFAULT_FROM: "{{ .Values.ticketsEmailAddress }}"
-  ACTION_MAILER_DEFAULT_TO: "{{ .Values.ticketsEmailAddress }}"
-  STRIPE_PUBLIC_KEY: "{{ .Values.stripePublicKey }}"
-  WEB_CONCURRENCY: "{{ .Values.pumaConcurrency }}"
-  RAILS_MAX_THREADS: "{{ .Values.pumaThreads }}"
+  {{ .Values.config | nindent 2 }}
+{{- if .Values.secret.create }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ .Release.Name }}-secrets
-type: Opaque
+{{- if .Values.secret.name }}
```

Comment:
No idea what's happening here :)

Reply:
I'm allowing for a created Secret to be named how ever you want it to be. In reality I'm trying to avoid creating Secrets altogether in favor of the external secrets operator.

This is here for some semblance of backwards compatibility where I allow for the helm values to not specify the Secret name and it'll do what it did before.

Reply:
These look good!